.github/workflows/test-python-sqlalchemy.yml

```diff
@@ -0,0 +1,62 @@
+name: Python SQLAlchemy
+on:
+
+  pull_request:
+    branches: ~
+    paths:
+    - '.github/workflows/test-python-sqlalchemy.yml'
+    - 'by-language/python-sqlalchemy/**'
+    - 'testing/ngr.py'
+  push:
+    branches: [ main ]
+    paths:
+    - '.github/workflows/test-python-sqlalchemy.yml'
+    - 'by-language/python-sqlalchemy/**'
+    - 'testing/ngr.py'
+
+  # Allow job to be triggered manually.
+  workflow_dispatch:
+
+# Cancel in-progress jobs when pushing to the same branch.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  test:
+    name: "CrateDB: ${{ matrix.cratedb-version }}
+     Python: ${{ matrix.python-version }}
+     on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-latest" ]
+        cratedb-version: [ "nightly" ]
+        python-version: [ "3.11" ]
+
+    services:
+      cratedb:
+        image: crate/crate:nightly
+        ports:
+          - 4200:4200
+          - 5432:5432
+
+    steps:
+
+      - name: Acquire sources
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+          cache: 'pip'
+          cache-dependency-path: |
+            by-language/python-sqlalchemy/requirements.txt
+            by-language/python-sqlalchemy/requirements-dev.txt
+
+      - name: Validate by-language/python-sqlalchemy
+        run: |
+          python testing/ngr.py --accept-no-venv by-language/python-sqlalchemy
```

Comment:
We finally added a universal test runner, effectively just wrapping a few other calls, to be able to maintain a concise incantation syntax.